src/pages/ApiParticulier.js

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Form from '../components/Form';
 import Nav from '../components/Nav';
-import ApiParticulierFormConfiguration from '../components/form/config/api-particulier';
+
+// Description du contexte
```

Comment:
I do not think these comments are useful as they are kind of a paraphrase of the variables themselves. Furthermore, they are additional lines that may be copied within all future forms, making this even more hard to maintain. I'd rather clarify the name of the variable, for instance : `IntroDescription` => `ContextDescription`.

Reply:
I think these comments are useful so that someone who needs to create a new API has a complete view of the customer experience they are creating.